doc/tutorials/lennard_jones/lennard_jones.ipynb

```diff
@@ -584,7 +584,10 @@
    "source": [
     "### Choosing the thermodynamic ensemble, thermostat\n",
     "\n",
-    "Simulations can be carried out in different thermodynamic ensembles such as NVE (particle __N__umber, __V__olume, __E__nergy), NVT (particle __N__umber, __V__olume, __T__emperature) or NpT-isotropic (particle __N__umber, __p__ressure, __T__emperature).\n",
+    "Simulations can be carried out in different thermodynamic ensembles such as\n",
+    "NVE (particle **N**umber, **V**olume, **E**nergy),\n",
+    "NVT (particle **N**umber, **V**olume, **T**emperature) or isotropic\n",
+    "NpT (particle **N**umber, **p**ressure, **T**emperature).\n",
```

Comment:
Note: although the double underscore syntax is part of the Markdown standard for making text boldface, the double asterisk syntax is the recommended version when making the first letter of a word bold (see sections on [Bold Best Practices](https://www.markdownguide.org/basic-syntax/#bold-best-practices)). Several Markdown processors actually don't process double underscores inside a word, and this is an issue for example in JupyterLab, the successor of Jupyter. In an upcoming PR, I will add support for JupyterLab in ESPResSo.